### ISSUE TITLE
All Images have no alt text

### ISSUE THREAD

**Issue by hyejida** [hyejida/fixing_the_bugs#3]
Alt text is critical for those with disabilities because it describes the function and appearance of a photo or graphic users cannot see. Therefore, I will add all alt text for images on my website.
